privaterelay/urls.py

```diff
@@ -32,7 +32,7 @@
     path('fxa-rp-events', views.fxa_rp_events),
     path('metrics-event', views.metrics_event),
 
-    path('accounts/profile/', views.profile, name='profile'),
+    # path('accounts/profile/', views.profile, name='profile'),
```

Comment:
Whoops, I'll have to remove this still - was moved to `if not settings.SERVE_REACT` below.

Reply:
Done in e7e8c775.